### DISCUSSION TITLE
Color content only

### DISCUSSION THREAD

**Discussion by Erimus-Koo** [Textualize/rich#3120] [Q&A] [answered]
Can I get the colored text only, without print and new lines.
e.g. 

```
from rich import print

abc = {
    "name": 'Erimus',
    "age": 7,
    "gender": "♂ male " * 100,
    "hobby": ["🍰", "⚽", "🍳", "🎱", "📖"],
}

print(abc)
```

<img width="709" alt="image" src="https://github.com/Textualize/rich/assets/17139670/22f1b246-2c4c-4532-befc-46e95936b550">

The output is very nice, but I'd like to get text with ANSI instead of printing them directly, and no line breaks for long text (line breaks for items in JSON are still required) for other subsequent operations.

**Comment by willmcgugan** [comment, marked as answer]
Use `console.print`, set `crop=False, overflow="ignore"`

To get the ansi sequences, search the docs for the "capture API".

**Reply by Erimus-Koo** [reply]
Thank you, that works.
